src/teb_config.cpp

```diff
@@ -84,6 +84,7 @@ void TebConfig::loadRosParamFromNodeHandle(const ros::NodeHandle& nh)
   nh.param("is_footprint_dynamic", robot.is_footprint_dynamic, robot.is_footprint_dynamic);
   nh.param("use_proportional_saturation", robot.use_proportional_saturation, robot.use_proportional_saturation);
   nh.param("transform_tolerance", robot.transform_tolerance, robot.transform_tolerance);
+  nh.param("max_vel_linear", robot.max_vel_linear, robot.max_vel_linear);
```

Comment:
Put together with the other vel params, and always in a fixed order (e.g. after max_vel_y and before max_vel_theta)